nixos/tests/ldap.nix

```diff
@@ -28,17 +28,16 @@ let
       users.ldap.daemon = {
         enable = useDaemon;
         rootpwmoddn = "cn=admin,${dbSuffix}";
-        rootpwmodpw = "/etc/nslcd.rootpwmodpw";
+        rootpwmodpwFile = "/etc/nslcd.rootpwmodpw";
       };
-      # NOTE: password stored in clear in Nix's store, but this is a test.
```

Comment:
This still applies

Reply:
yes, but this NOTE wasn't next to the password file being written, and the same NOTE exists a bit further down too.

I moved the `environment.etc."nslcd.rootpwmodpw".source =` below `environment.etc."ldap/bind.password".source`, and kept only one NOTE.